tests/test_app/TestApp/Controller/DependenciesController.php

```diff
@@ -36,4 +36,9 @@ public function requiredDep(stdClass $dep, $any = null, ?string $str = null)
     {
         return $this->response->withStringBody(json_encode(compact('dep', 'any', 'str')));
     }
+
+    public function variadic()
```

Comment:
I assume it used to work if you had `variadic(...$params)`?

Reply:
I'll add a test for that as well.